packages/functions/src/api/machine.ts

```
@@ -0,0 +1,224 @@
+import {z} from "zod"
+import {Hono} from "hono";
+import {notPublic} from "./auth";
+import {Result} from "../common";
+import {describeRoute} from "hono-openapi";
+import {assertActor} from "@nestri/core/actor";
+import {Realtime} from "@nestri/core/realtime/index";
+import {validator} from "hono-openapi/zod";
+import {CreateMessageSchema, StartMessageSchema, StopMessageSchema} from "./messages.ts";
+
+export module MachineApi {
```

Biome (1.9.4)

[error] Use the namespace keyword instead of the outdated module keyword.

The module keyword is deprecated to avoid any confusion with the ECMAScript modules which are often called modules.
Safe fix: Use namespace instead.


(lint/suspicious/useNamespaceKeyword)